docs/en-US/basic-usage.md

```diff
@@ -0,0 +1,35 @@
+# Basic Usage
+
+## Using a GUI
+
+This documentation assumes some very basic terminal/command line knowledge to run simple commands. However, some people prefer the convenience of a visual UI. If you fall into this category then consider the [Vagrant Manager](http://vagrantmanager.com/)  project.
+
+Note: Until you provision VVV for the first time, Vagrant Manager will not pick up VVV. Running `vagrant up --provision`  inside the VVV folder and allowing it to succesfully finish should be enough.
+
+## Turning VVV On
+
+```
+vagrant up
+```
+
+If Vagrant triggers are installed, and the VVV machine is turned off, this will also run the provisioner.
+
+## Turning VVV Off
+
+```
+vagrant halt
+```
+
+This will shut down the virtual machine. If the machine is frozen for whatever reason, add the ` --force` parameter. If it still refuses to power off, open Virtualbox and manually power the VM off.
+
+## Restarting VVV
+
+```
+vagrant reload
+```
+
+This will do a restart of the Virtual Machine, and is the same as running `vagrant halt;vagrant up`
+
+## Reloading `vvv-custom.yml`
+
+If you make any changes to your config file, they won't take immediate effect. For changes to take hold, restart VVV using `vagrant reload`
```

Comment:
Fixed in https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1112/commits/1ce00fef55e332c8fdd7e7a4bd159acdc6b72f2b